osu.Game/Localisation/ModPresetColumnStrings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class ModPresetColumnStrings
```

Comment:
I think I'd just put these in `ModSelectOverlayStrings` to keep things in one place.